src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: {
+    item: [],
+    status: 'Under Construction',
+  },
+  reducers: {
+    showCategories: (state) => state.status,
+  },
```

Comment:
- [ ] The "Under construction" status should come from an action, as later on it will be triggered by the button you have in your categories page, please consider refactoring your code so you change the state based on a dispatched action! 🤓 🔧

Example: 

```js

    checkStatus: (state, action) => {
      state.item =
        action.payload === "Under construction"
          ? "Under construction"
          : state.item
    },
```